### PR TITLE
Business alert rest test

### PR DESCRIPTION
**Pull request by jlschatz**

### CODE REVIEW

File: rest/AppDynamics.go [hunk 1]
```diff
@@ -42,6 +42,19 @@ func receiveAppDynamicsAlert(w http.ResponseWriter, r *http.Request) {
...
 
+func receiveNonTechAppDynamicsAlert(w http.ResponseWriter, r *http.Request) {
+
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		service.SendError(err)
+		return
+	}
+
+	database.SaveAudit("APPDYNAMICS", string(body))
```
Comment (zamedic) [inline rest/AppDynamics.go]: Not App Dynamics - its more Non Technical

File: rest/AppDynamics.go [hunk 2]
```diff
@@ -42,6 +42,19 @@ func receiveAppDynamicsAlert(w http.ResponseWriter, r *http.Request) {
...
+	database.ReceiveAppynamicsMessage()
```
Comment (zamedic) [inline rest/AppDynamics.go]: Again, not app dynamics

File: rest/rest.go [hunk 3]
```diff
@@ -50,7 +50,7 @@ func getRouter() *mux.Router {
 	r.HandleFunc("/appdynamics/system/queue", addAppdynamicsQueueEndpoint).Methods("POST")
 	r.HandleFunc("/appdynamics/system", addAppDynamicsEndpoint).Methods("POST")
 	r.HandleFunc("/appdynamics/execute", executeCommandFromAppdynamics).Methods("POST")
-	r.HandleFunc("/appdynamics/alert/nontech", businessAppDynamicsAlert).Methods("POST")
+	r.HandleFunc("/appdynamics/alert/nontech", receiveNonTechAppDynamicsAlert).Methods("POST")
```
Comment (zamedic) [inline rest/rest.go]: Would rather decouple from app dynamics, since it a simple string which can come from anywhere

**Comment by zamedic** [comment]
Can this one be deleted?

**Comment by jlschatz** [comment]
It can my good sir.

**Pull request closed without merging**